courses/free-courses-bn.md

```diff
@@ -274,6 +274,7 @@ Online School
 * [Python All You Need To Know](https://www.youtube.com/playlist?list=PL_XxuZqN0xVCH8fRfZ8sUuKEQQYIFMjtJ) - Stack Learner
 * [Python Bangla Tutorial 2023](https://www.youtube.com/playlist?list=PLNMnAEqLBwmpR8JDBOEl0jrzmH1vPnO7v) - Hablu Programmer
 * [Python Bangla Tutorials for Beginners](https://www.youtube.com/playlist?list=PLgH5QX0i9K3rz5XqMsTk41_j15_6682BN) - Anisul Islam
+* [Python Bangla Tutorial & Data Science Full Course || Full Guidelines for Python, ML, AI, DL & Django](https://www.youtube.com/watch?v=A3FKSsYy0Zg&list=PLKdU0fuY4OFcot0zyVbM1-zKf_eCUK4zQ) - STUDY MART
```

Comment:
For YouTube, we prefer "playlist" links, not individual video links. 

The link you provided seems different from the title. 

Are you referring to this link: https://www.youtube.com/playlist?list=PLKdU0fuY4OFcot0zyVbM1-zKf_eCUK4zQ    ? 

Additionally, use the title from the resource and replace "|" with "-".

Reply:
fixed please approve it is in order now